interactions/client/context.py

```diff
@@ -139,6 +140,8 @@ async def send(
         :type components?: Optional[Union[ActionRow, Button, SelectMenu, List[Union[ActionRow, Button, SelectMenu]]]]
         :param ephemeral?: Whether the response is hidden or not.
         :type ephemeral?: Optional[bool]
+        :param suppress_embeds: Whether embeds are shown in the message.
```

Comment:
Actually this makes more sense, sorry for being nitpicky 
```suggestion
        :param suppress_embeds: Whether embeds are not shown in the message.
```